PsychSourceGL/Source/Common/Eyelink/EyelinkImageTransfer.c

```diff
@@ -1,294 +1,358 @@
-/*
```

Comment:
Had included our version in the PR only because I'd already written it and tested it to be working. Other good points you've made though. Reverting to your previous version failed to work as intended, and I'm debugging. Later commit will either resolve the issues in your previous version, or adapt improvements into ours.

Reply:
resolved in 8b8988d501a4a14a192342c25ed34b88c7dd787b. restore version from fe6b243bdcf7d595530d1ea53b3a004c2a183ffd and resovled Host PC error related to unsupported use of negative height for vertical flip in bitmap metadata